src/Symfony/Bridge/Twig/Extension/ContainerAwareRuntimeLoader.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Loads Twig extension runtimes.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class ContainerAwareRuntimeLoader implements \Twig_RuntimeLoaderInterface
+{
+    private $container;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load($name)
+    {
+        $id = 'twig.extension.runtime.'.$name;
```

Comment:
please don't force this convention: it forbids third-party bundles to use this behavior while following best practices for shared bundles. The id of the runtime should be provided as an optional attribute in the ``twig.extension`` tag IMO, to build a mapping between extension names and runtimes